packages/awesomeshot/build.sh

```diff
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://github.com/mayTermux/awesomeshot
 TERMUX_PKG_DESCRIPTION="A command-line screenshot tool written in bash"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=1.0.5
+TERMUX_PKG_VERSION=1.1.1
```

Comment:
1.1.1 is pre-release, downgrade to 1.1.0

Reply:
> 1.1.1 is pre-release, downgrade to 1.1.0

sorry, why can't update to pre-release? before i pull request pre-release too https://github.com/termux/termux-packages/pull/10060

Reply:
Because auto update system won't recognise it as an update (as I have asked you before to enable auto update)

Reply:
> Because auto update system won't recognise it as an update (as I have asked you before to enable auto update)


I have set `TERMUX_PKG_AUTO_UPDATE=true` and downgrade version from `1.1.1 pre-release` to `1.1.0` on the latest commit